scripts/collect-modules.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Print commands before executing them (useful for troubleshooting)
+set -x
+
+if [[ "$CONFIGURATION" = *Debug* ]]; then
+  echo "Debug build. Modules are not collected."
+  exit 0
+fi
+
+if [[ -z "$CONFIGURATION_BUILD_DIR" ]]; then
+  echo "Missing env CONFIGURATION_BUILD_DIR" 1>&2
+  exit 1
+fi
+
+if [[ -z "$UNLOCALIZED_RESOURCES_FOLDER_PATH" ]]; then
+  echo "Missing env UNLOCALIZED_RESOURCES_FOLDER_PATH" 1>&2
+  exit 1
+fi
+
+if [[ -z "$DERIVED_FILE_DIR" ]]; then
+  echo "Missing env DERIVED_FILE_DIR" 1>&2
+  exit 1
+fi
+
+nodePath="node"
+thisFilePath=$(dirname $0)
+collectModulesScript="$thisFilePath/../dist/js/tools/collectModules.js"
+
+destination="$CONFIGURATION_BUILD_DIR/$UNLOCALIZED_RESOURCES_FOLDER_PATH"
+modulesOutput="$destination/modules.json"
+if [[ -z "$SOURCE_MAP_PATH" ]]; then
+  sourceMap="$DERIVED_FILE_DIR/main.jsbundle.map"
+else
+  sourceMap="$SOURCE_MAP_PATH"
+fi
+if [[ -z "$MODULES_PATHS" ]]; then
```

Comment:
L: Is there more than one path? otherwise `MODULES_PATH`

Reply:
Yes, there can be more, most of the time it will be just `node_modules`, but for example our sample would not work with just one path.